File: face-check/SecurityDomainModule/src/main/java/com/zikpak/facecheck/requestsResponses/workSite/selectWorkSite/SelectWorkSiteResponse.java
package com.zikpak.facecheck.requestsResponses.workSite.selectWorkSite;


import lombok.Builder;
import lombok.Data;

@Data
@Builder
public class SelectWorkSiteResponse {

    private String selectedWorkSiteName;
    private Integer selectedWorkSiteId;
    private Integer workerId;


}

File: face_check/lib/api_client/model/select_workSite_response.dart
class SelectWorkSiteResponse {
  final String selectedWorkSiteName;
  final int selectedWorkSiteId;
  final int workerId;

  SelectWorkSiteResponse({
    required this.selectedWorkSiteName,
    required this.selectedWorkSiteId,
    required this.workerId,
  });

  factory SelectWorkSiteResponse.fromJson(Map<String, dynamic> json) {
    return SelectWorkSiteResponse(
      selectedWorkSiteName: json['selectedWorkSiteName'],
      selectedWorkSiteId: json['selectedWorkSiteId'],
      workerId: json['workerId'],
    );
  }

  Map<String, dynamic> toJson() {
    return {
      'selectedWorkSiteName': selectedWorkSiteName,
      'selectedWorkSiteId': selectedWorkSiteId,
      'workerId': workerId,
    };
  }
}